### DISCUSSION TITLE
Major Updates & Announcements

### DISCUSSION THREAD

**Discussion by SilasK** [metagenome-atlas/atlas#417] [Announcements]
Follow this discussion to be informed about major updates to Metagenome-atlas
I will post information about **New features, Tutorials & Courses**.

Simply click on *Subscribe* on the right-hand side. 

For non-maintainers, please start a new discussion for your question in order not to spam all the listeners here.

**Comment by SilasK** [comment]
@ctb @mdehollander   @botellaflotante @Sofie8 @molecules  @jmtsuji @francis29029 @smcolby  @SF-Dragon @wangzhichao1990 @NickiShaw @colinbrislawn @slambrechts @ChristianFurbo @makrez @masun  @soilmicrobiome @metagenome-atlas/eukaryotes @coverall2357 @raw937 @taylorreiter @vmikk @animesh @arsuke @JLammons14 @zztin @laineybright @rbis3636 @alexhbnr @johanneswerner @CedricMidoux @luke-dt @kazumaxneo @anitajnorman @MalbertR @Jiung-Wen @aduvermy @ninashenker @jonnatauriainen @boaty @panrenzaishu @jjsanchezgil @pbaCamille @cslamo @bhimbbiswa @Ahmed-Shibl @ejongepier @gruningerrj @bramvandijk88 @skembel @ShirleyCXiao 


Hey all. I added you to this GitHub thread where I will post major updates about **metagenome-atlas**. I will post information about New features, Tutorials & Courses. You shouldn't get too many messages. 

- If you used metagenome-atlas for your publication add them to our [Hall of Fame](https://github.com/metagenome-atlas/atlas/discussions/421) 🥇 .
- If you are still using metagenome-atlas, let us know [who you are and what you plan to do with metagenome-atlas. #422

**Comment by SilasK** [comment]
@AlexanderBartholomaeus @maudimi @baptisteavot @jfourquet2 @JesseKerkvliet @MaxRubinBlum @ganiatgithub @srikanthkris @canaanwc @mmanno @adscheah @LeeBergstrand @noirot @fconstancias @Annie-msi @Leitemfa @spleonard1 @telatin @tianlei22 @mradz19 @jtamames @ForrestBurt @mdjlynch @fetyj @boulund @bobems1215 @IRSADFERHAT @McGlock @JingqiuLiao @razz1618 


Hey all. I added you to this GitHub thread where I will post major updates about **metagenome-atlas**. I will post information about New features, Tutorials & Courses. You shouldn't get too many messages. 

- If you used metagenome-atlas for your publication add them to our [Hall of Fame](https://github.com/metagenome-atlas/atlas/discussions/421) 🥇 .
- If you are still using metagenome-atlas, let us know [who you are and what you plan to do with metagenome-atlas. #422

**Comment by SilasK** [comment]
Three-day course organized online on 27th September, 1st October, and 4th October 2021
Metagenome-atlas: Three commands to start analysing your metagenome data.

More information on the course webpage :  https://www.oulu.fi/uniogs/node/212131
The course is funded by University of Oulu Graduate School, TNS-DP

Registration before 10th September via https://link.webropolsurveys.com/S/6F8ED27A95D4D331. A short motivation letter attached with your registration is mandatory.

**Comment by SilasK** [comment]
MEtagenome-atlas 2.8 is out #464

**Reply by jmtsuji** [reply]
Wow, this is amazing work! Big thanks to you, @yanhui09 , @psj1997 , and others who contributed.

**Comment by animesh** [comment]


**Reply by SilasK** [reply]
It's on conda. So you have jus to insist a bit more: `mamaba install metagenome-atlas=2.8`

**Comment by SilasK** [comment]
[Atlas v 2.9](https://github.com/metagenome-atlas/atlas/releases/tag/v2.9.0) is out. 

The major feature is the new `atlas init-public` command that allows you to create an atlas project from public data in the NCBI SRA. The reads are downloaded using the most efficient way that I found. The download process can be stopped and restarted, if necessary. 

We have also first support for strain resolution data using inStrain.

**Comment by animesh** [comment]


**Reply by SilasK** [reply]
The version should bebtakeb automatically from the github tag. Ibassume you didn't manage to install the newest version. 
Check 'conda list metagenome-atlas'

I reccomend to install atlas as
'mamba install metagenome-atlas>=2.9'

**Comment by animesh** [comment]


**Reply by SilasK** [reply]
Hmm maybe it should be 'mamba install metagenome-atlas=2.9' 

Sorry for the inconvinience.

**Comment by colinbrislawn** [comment]


**Reply by SilasK** [reply]
@colinbrislawn You are right. I kindly ask users with problems to create new issues.

**Comment by SilasK** [comment]
Atlas v2.14 uses checkm2 !

**Comment by SilasK** [comment]
Atlas v2.18 is out. It uses co-binning now as default, with an automatic setting of BinGroup and Binner depending on data size.
It also uses skani for dereplication and produces more useful bin reports already with the number of dereplicated species. 

[Release](https://github.com/metagenome-atlas/atlas/releases/tag/v2.18.0)
[Blog_post](https://silask.github.io/post/cobinning/)

Tell me what you think?

**Comment by SilasK** [comment]
I improved the tutorial R code on how to use the Genecatalog extensively.
It contains how to access a subset of genes from the hdf file. e.g. genes annotated with Kegg.
And how to do different abundance tests with these.

https://github.com/metagenome-atlas/Tutorial/blob/master/R/Analyze_genecatalog.Rmd
